client/src/test/java/io/hstream/HStreamClientTest07.java

```diff
@@ -463,12 +463,9 @@ void shardBalance() throws Exception {
     Assertions.assertEquals(readResAsSet, writeResAsSet);
 
     // 3. Assert the union of keys all consumers own is equal to all keys
-    HashSet<String> expectedKeys = new HashSet<>();
-    for (int i = 0; i < shardCount; ++i) {
-      expectedKeys.add("key-" + i);
-    }
-
+    HashSet<String> expectedKeys = new HashSet<>(writeRes.keySet());
     Assertions.assertEquals(unionOfKeys, expectedKeys);
+
     // 4. Assert the keys any two consumers own is disjoint
```

Comment:
This assertion is also not valid：a key can be assigned to a consmer firstly and reassigned to another later.